test/powershell/Language/Scripting/TestsOnWinFullOnly.ps1

```diff
@@ -0,0 +1,27 @@
+﻿function Run-TestOnWinFull
+{
+    [CmdletBinding()]
+    param( [string]$name )
+
+    switch ($name) 
```

Comment:
fixed

---
In reply to: [76494845](https://github.com/powershell/powershell/pull/2086#discussion_r76494845) [](ancestors = 76494845)